scripts/smeshr/MODIS-STC/MODIS_albedo_basin.sh

```diff
@@ -102,7 +102,7 @@ modis_basin() {
 }
 export -f modis_basin
 echo "Extracting MODIS albedo"
-${PARALLEL_call} modis_basin ::: ${1}/wy${2}/*.tif
+${PARALLEL_call} modis_basin ::: ${1}/wy${2}/*.tif $BASIN_DOMAIN
```

Comment:
Just FYI - If you use an `export` like seen on line 37 or 38, then you don't need to pass this argument to the `parallel` command.

[Docs for variables in parallel](https://www.gnu.org/software/parallel/parallel_tutorial.html#transferring-environment-variables-and-functions)

Reply:
Updated in [latest commit](https://github.com/UofU-Cryosphere/isnoda/pull/64/commits/97250ea6315b0e2cf46c03974824d17ea8337f76)